src/components/AsideBar/AsideBar.jsx

```diff
@@ -21,6 +23,22 @@ const AsideContainer = styled.aside`
     gap: 32px;
   }
 `;
+const BoxShadow = styled.div`
+  position: absolute;
```

Comment:
ten rozdzielacz to powinien być border-right i zwykły box shadow na aside, który pada 1px w prawo, 
sam border ma chyba 2px i box shadow wystaje 1px

Reply:
Gash w tabeli ma 2 kolory. Ja się dopatrzyłem 3 różnych na belce. Jak zrobię to box shadow to nie będzie tak wyglądać.

Reply:
![image](https://github.com/pawelrogowski/goit-wallet-app-03-frontend/assets/110867327/f255cc75-b111-4f9e-8572-9d93e84e4e82)
2px border i 1 px shadow